90zfsbootmenu/zfsbootmenu-preinit.sh

```diff
@@ -24,6 +24,7 @@ mkdir -p "${BASE}"
 # shellcheck disable=SC2154
 cat >> "/etc/profile" <<EOF
 # Added by zfsbootmenu-preinit.sh
+export BASE="/zfsbootmenu"
```

Comment:
What do you think about changing this to `ZBMROOT` or something less generic than `BASE`?

Reply:
`ZBMROOT` is much more meaningful than `BASE` - I'd be glad to see that globally changed if/after this PR is merged.